webrtc-stats.html

```diff
@@ -515,6 +516,17 @@ <h3>
               accessed by the <code><a>RTCRemoteOutboundRtpStreamStats</a></code>.
             </p>
           </dd>
+          <dt>
+            <dfn><code>media-source</code></dfn>
+          </dt>
+          <dd>
+            <p>
+              Statistics for the media produced by a <code>MediaStreamTrack</code> that is currently
```

Comment:
It should be clear that the data here is input to the encoder after gUM constraints are applied and these are not raw media source metrics.

Reply:
Done